packages/components/src/navigator/stories/index.tsx

```diff
@@ -232,3 +232,64 @@ function ProductDetails() {
 		</Card>
 	);
 }
+
+const NestedNavigatorTemplate: ComponentStory< typeof NavigatorProvider > = ( {
+	style,
+} ) => (
+	<NavigatorProvider
+		style={ { ...style, height: '100vh', maxHeight: '450px' } }
+		initialPath="/"
+	>
```

Comment:
It's good practice for our stories to pass the `props` down to `NavigatorProvider`, instead of hardcoding them. This plays nicely with Storybook's controls too.

```suggestion
const NestedNavigatorTemplate: ComponentStory< typeof NavigatorProvider > = ( {
	style,
	...props
} ) => (
	<NavigatorProvider
		style={ { ...style, height: '100vh', maxHeight: '450px' } }
		{ ...props }
	>
```

At the bottom of the file, we can then add a `NestedNavigator.args` object with the `initialPath` prop.

I would actually argue that a better way to showcase the `goToParent` functionality is to have an `initialPath` different from the root:

```
NestedNavigator.args = {
	initialPath: '/child2/grandchild',
};
```